src/pkg/auth/auth.go

```diff
@@ -63,6 +63,15 @@ var (
 	authTemplate = template.Must(template.New("auth").Parse(authTemplateString))
 )
 
+type ErrNoBrowser struct {
+	Err error
+	URL string
+}
+
+func (e ErrNoBrowser) Error() string {
+	return fmt.Sprintf("failed to open browser: %v. Please open the following URL in your browser: %s", e.Err, e.URL)
```

Comment:
I wonder if we should say "a browser". For example, the user could log in on their phone, right?

```suggestion
	return fmt.Sprintf("failed to open browser: %v. Please open the following URL in a browser to login: %s", e.Err, e.URL)
```